app/controllers/VATRegistrationExceptionController.scala

```diff
@@ -60,13 +60,13 @@ class VATRegistrationExceptionController @Inject()(mcc: MessagesControllerCompon
       formProvider().bindFromRequest().fold(
         (formWithErrors: Form[_]) =>
           Future.successful(BadRequest(view(formWithErrors, NormalMode))),
-        value =>
+        value => {
           sessionService.save[Boolean](VATRegistrationExceptionId.toString, value).map { cacheMap =>
             if (value) {
```

Comment:
I think this should also only do the TM update if the exception/exemption FS is disabled